content/docs/en/elements/components/button.md

```diff
@@ -3,7 +3,7 @@ title: Button
 apiRef: https://docs.nativescript.org/api-reference/classes/_ui_button_.button
 ---
 
-The Button component is used to display a button.
+The Button component is used to display a button that can react to a tap event.
```

Comment:
This is fine, but I wonder if we can improve on this (in the future), since it can react to multiple gestures, not just the `tap` gesture.

But leave it like this for now.

Reply:
It's a difficult one, because you don't want to over-complicate it for first time users - but you also don't want to neglect possible feature attributes or events...